src/drawing/DrawingPathGroup.scss

```diff
@@ -0,0 +1,12 @@
+$ba-DrawingTarget-transitionDelay: 25ms;
+$ba-DrawingTarget-transitionDuration: 200ms;
+
+.ba-DrawingPathGroup-decoration {
+    opacity: 0;
+    transition: opacity $ba-DrawingTarget-transitionDuration ease;
+}
+
+.ba-DrawingPathGroup.is-active .ba-DrawingPathGroup-decoration {
```

Comment:
This could be converted to the following to cut down on the repetition:

```
.ba-DrawingPathGroup-decoration {
    opacity: 0;
    transition: opacity $ba-DrawingTarget-transitionDuration ease;

    .is-active & {
        opacity: 1;
        transition-delay: $ba-DrawingTarget-transitionDelay;
    }
}